ci/Jenkinsfile-release

```diff
@@ -367,10 +374,10 @@ def publishCondaArtifact(arch, pkgName) {
 }
 
 
-def getCondaPkgName(params, pyVersion) {
+def getCondaPkgName(pyVersion, packageName) {
     return sh(returnStdout: true, script:
             """
-            CONDA_PKG_CURRENT_ARCH_PATH=\$(conda build h2o_pysparkling_${params.spark} --py ${pyVersion} --output-folder "." --output | tail -1)
+            CONDA_PKG_CURRENT_ARCH_PATH=\$(conda build ${packageName} --py ${pyVersion} --output-folder "." --output | tail -1)
```

Comment:
spark version removed from package name

Reply:
see the name entry in packages: https://github.com/h2oai/sparkling-water/pull/2426/files/abbb67db0ab47caa7ed73b3353aa2941ef384ad3#diff-7178e3aa434d15a33bacb11e21cf4af3ce2f2bb6fb692536f75545dc247a8787R280